src/components/MoveAdder.test.ts

```diff
@@ -0,0 +1,86 @@
+import { IMove } from "battlemovr";
+import { expect } from "chai";
+import { stubBlankGame } from "../fakes.test";
+import { IPokemon } from "./Battles";
+import { MoveAdder } from "./MoveAdder";
+
+describe("MoveAdder", () => {
+
+    it("adds/replaces a new move to a Pokemon's moveset when the move index is valid", (): void => {
+        //Arrange
```

Comment:
Nit: everywhere else, there's a space after `//` and before `Arrange`/`Act`/`Assert`. Please keep that convention here.